op-node/metrics/metrics.go

```diff
@@ -528,12 +528,10 @@ func (m *Metrics) RecordSequencerSealingTime(duration time.Duration) {
 // The server will be closed when the passed-in context is cancelled.
 func (m *Metrics) Serve(ctx context.Context, hostname string, port int) error {
 	addr := net.JoinHostPort(hostname, strconv.Itoa(port))
-	server := &http.Server{
-		Addr: addr,
-		Handler: promhttp.InstrumentMetricHandler(
-			m.registry, promhttp.HandlerFor(m.registry, promhttp.HandlerOpts{}),
-		),
-	}
+	server := server.NewHttpServer(promhttp.InstrumentMetricHandler(
```

Comment:
There is variable shadowing here, with `server` being both a variable and a package name. Perhaps we either import the package as `op_server` or call the variable here something else

Reply:
Fixed by renaming module.